.github/workflows/tests.yaml

```diff
@@ -156,12 +141,11 @@ jobs:
         node-version: [14, 16]
 
     env:
-      COGNITE_PROJECT: platypus
-      COGNITE_BASE_URL: https://greenfield.cognitedata.com
-      COGNITE_CREDENTIALS: ${{ secrets.COGNITE_CREDENTIALS }}
+      COGNITE_PROJECT: cognitesdk-js
+      COGNITE_BASE_URL: https://api.cognitedata.com
       COGNITE_CLIENT_SECRET: ${{ secrets.COGNITE_CLIENT_SECRET }}
-      COGNITE_CLIENT_ID: ${{ secrets.COGNITE_CLIENT_ID }}
-      COGNITE_AZURE_TENANT_ID: ${{ secrets.COGNITE_AZURE_TENANT_ID }}
+      COGNITE_CLIENT_ID: cb491d7a-c346-4910-9a10-1650df34fbbb
+      COGNITE_AZURE_TENANT_ID: cogniteappdev.onmicrosoft.com
```

Comment:
Should we rename this? I see microsoft uses the term domain in azure for this, and they use the term tenant id for an id like this `267cfdd8-a207-4320-80f2-a4352b15048f`